### ISSUE TITLE
Is cargo gen alias working as expected?

### ISSUE THREAD

**Issue by MarcoIeni** [cargo-generate/cargo-generate#268]
If you install `cargo generate` with `cargo install generate`, then `cargo gen` doesn't work (at least for me).
Instead if I run `cargo run -- gen` it works as expected.

Am I missing something? Since the majority of the users are installing `cargo generate` by using `cargo install` shouldn't we just remove the `cargo gen` alias? It might be confusing.

**Comment by k0pernicus** [comment]
Thanks @MarcoIeni for reporting this issue.

I don't know if this is an issue with the `structopt` crate or a more general `cargo` subcommand issue to be honest 🤔 

I tried to revert the recent changes in `main.rs` and `lib.rs` (to know if the `visible_alias` directive is not buggy), and this did not changed the behaviour...  
Then, I reverted the `structopt` crate version from `0.3.15` to `0.2.15` and no change at all...

However, adding manually the alias in `$HOME/.cargo/config` works:  
```toml
[alias]
gen = "generate"
```

I tested an older version of the build (https://github.com/ashleygwilliams/cargo-generate/tree/v0.4.0) to test if we have a regression issue somewhere, and it worked well

```
➜  cargo-generate git:(975ea2c) cargo gen --version
cargo-generate 0.4.0
```

I don't think it's an issue of structopt, as I tested a local install using structopt `0.2.15` (the same than our 0.4.0 version), but the `cargo` crate changed from `0.30` to `0.40`... maybe a compatibility issue registering the aliases from `structopt`?

I checked quickly in the `cargo` crate but did not found something very interesting there (https://github.com/rust-lang/cargo/issues?q=add+alias+).

I will fill an issue in `structopt` to know if they are aware about this, and if they have a solution too (maybe registering the alias somewhere?).
